creator-node/src/ImageProcessingQueue.js

```diff
@@ -97,4 +101,4 @@ class ImageProcessingQueue {
   }
 }
 
-module.exports = new ImageProcessingQueue()
+module.exports = ImageProcessingQueue
```

Comment:
why this change

Reply:
This was changed so that it can be initialized in `ServiceRegistry` where I can get the `prometheusRegistry`

Reply:
oh right, duh